### PR TITLE
Add firefox/privacy/book.ftl

### PR DESCRIPTION
**Pull request by amychurchwell**

### CODE REVIEW

File: en/firefox/privacy/book.ftl [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+privacy-book-look-at-the = Look at the bigger picture:
+
+privacy-book-we-already-talked = We already talked about how people tend to like convenience – remember? When it comes to information we consume, it might be nice to just see our attitude and expectations confirmed because it’s pleasant and calming. But does that make the information accurate? Do we get the big picture or just a rather small extract?
+
+privacy-book-you-can-of = <strong>You can, of course, have your favorites</strong> but check out different outlets from time to time to learn about different opinions and give other perspectives a chance.
+
+privacy-book-ask-yourselves-do = <strong>Ask yourself:</strong> Do the different outlets report on the issues you care about? And is the information shared more or less consistent or does it differ a lot? That’s the best way to battle filter bubbles and, ultimately, also filter bubbles and, ultimately, also misinformation.
```
Comment (amychurchwell) [inline en/firefox/privacy/book.ftl:171]: [Feedback from Peiying](https://github.com/mozilla-l10n/www-l10n/pull/127#discussion_r523254885):
> This sentence needs a rewrite. The three components are not in parallel: That’s the best way to **battle** filter bubbles and, ultimately, also **filter** bubbles and, ultimately, also misinformation. Battle and the 2nd filter are verbs. There needs to be a verb for misinformation.

File: en/firefox/privacy/book.ftl [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+privacy-book-what-feels-right =  <strong>What feels right for you today might change tomorrow.</strong> And that's totally fine. It doesn't hurt to reevaluate your view on the analogue world and the web occasionally and adjust them if necessary.
+
+privacy-book-what-does-the = What does the world know about you?
+
+privacy-book-to-an-extent = To an extent, we can control what information about ourselves is available on the web. Unfortunately, this can’t protect us entirely against dangers. A particularly bad one is identity theft. Here’s what you might want to consider:
+
+privacy-book-search-your-name = <strong>Search your name online occasionally.</strong> This will give you an idea of what others find out if they’re interested in you, what information is already around, and what you might want to get removed.
+
+privacy-book-dont-have-time = <strong>Don’t have time?</strong> Set up a { -brand-name-google } Alert for your name and get notified immediately if something new pops up online.
```
Comment (amychurchwell) [inline en/firefox/privacy/book.ftl:109]: [Feedback from Peiying:](https://github.com/mozilla-l10n/www-l10n/pull/127#discussion_r523250209)
> if something new pops up online - what is this referring to? Other than breach alert, what kind of notification that is privacy related that would trigger popups?

**Comment by amychurchwell** [comment]
@rraue There are two comments on this copy from firefox/privacy/book that Peiying flagged as issues for localization. Would you be able to help us resolve? Perhaps the original copy writer can help us out?

File: en/firefox/privacy/book.ftl (hunk 1 above)
Comment (rraue) [reply en/firefox/privacy/book.ftl:171]: Good catch, seems like I hickup in editing. My bad. The sentence should be just easy:

"That’s the best way to battle filter bubbles and misinformation."

File: en/firefox/privacy/book.ftl (hunk 2 above)
Comment (rraue) [reply en/firefox/privacy/book.ftl:109]: This is not referring to pupups but if something new comes up, anything that is indexed by Google and includes your name. 

<strong>Don’t have time?</strong> Set up a { -brand-name-google } Alert for your name and get notified if you get mentioned on a website.

Comment (amychurchwell) [reply en/firefox/privacy/book.ftl:109]: Thanks for your help, Raphael!

**Comment by amychurchwell** [comment]
@peiying2 this PR should be up to date now. Let me know if we missed anything. 😄 thanks for your help.

File: en/firefox/privacy/book.ftl [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+privacy-book-de-blog-post = If you want to learn more about the latter, here’s <a href="https://blog.mozilla.org/firefox/how-to-spot-real-fake-news/">an interesting blog post</a> that can be helpful.
+
+privacy-book-disable-unnecessary-location = Disable (unnecessary) location services:
+
+privacy-book-todays-mobile-devices = Today’s mobile devices typically have the ability to determine, store and share their location. That’s convenient for navigation or when you’ve lost your phone. At the same time though, this quite sensitive piece of information may also be used by apps, even if you don’t want it.
+
+privacy-book-many-social-media = <strong>Decide case by case:</strong> Many social media apps offer to share your location when you post something. It’s up to you if you want others to know where you are and it’s totally fine to say no.
+
+privacy-book-for-each-of = <strong>Decide for each app:</strong> For each of the apps on your device that wants access to your location information, you can decide if you want to share it. All major operating systems allow for you to change your settings so that you limit access for individual or all apps.
```
Comment (peiying2) [inline en/firefox/privacy/book.ftl:182]: Remove `for` so it reads `... operating systems allow you to change your settings ...`

**Review by peiying2 — CHANGES_REQUESTED** [review]
@amychurchwell Thank you for the updates. I have verified that all the comments were addressed except one. Could you confirm this one was purposely left out?

Also, have you updated the config file so this page is only for de, fr. What about en-*? It needs to happen before the page goes to Pontoon.

**Comment by amychurchwell** [comment]
@peiying2 working on updating config now. Thank you for the reminder.

**Comment by amychurchwell** [comment]
@peiying2 the config update has merged!

**Review by peiying2 — APPROVED** [review]
Thank you for updating the config file. We are done!

**Pull request merged**